claripy/ast/bv.py

```diff
@@ -49,6 +51,65 @@ class BV(Bits):
             11111111111111111111111111111100
     """
 
+    SLE: ClassVar[Callable[..., Bool]]
+    __rrshift__: ClassVar[Callable[..., BV]]
```

Comment:
The original `__rrshift__` method is assigned dynamicly instead of a class method:
```
BV.__rrshift__ = operations.reversed_op(BV.__rshift__)
```
Currently, using `ClassVar` is the only way I have found that helps mypy to add type hinting without too much modification.